packages/search-input/src/SearchInput/SearchInput.story.tsx

```diff
@@ -0,0 +1,258 @@
+  },
+  {
+    name: 'fig',
+    description:
+      'A soft pear-shaped fruit with sweet dark flesh and many small seeds',
+  },
+  {
+    name: 'grape',
+    description:
+      'A berry, typically green, purple, red, or black, growing in clusters on a vine',
+  },
+  {
+    name: 'honeydew',
+    description:
+      'A melon of a variety with smooth pale skin and sweet green flesh',
+  },
+  {
+    name: 'iceberg-lettuce',
+    description:
+      'A lettuce of a variety having a dense round head of crisp pale leaves.',
+  },
+  {
+    name: 'jalapeño',
+    description:
+      'A very hot green chili pepper, used especially in Mexican-style cooking',
+  },
+  {
+    name: 'kiwi',
+    description: 'A fruit with a thin hairy skin, green flesh, and black seeds',
+  },
+  {
+    name: 'lemon',
+    description:
+      'A yellow, oval citrus fruit with thick skin and fragrant, acidic juice:',
+  },
+  {
+    name: 'melon',
+    description:
+      'The large round fruit of a plant of the gourd family, with sweet pulpy flesh and many seeds',
+  },
+  {
+    name: 'nectarine',
+    description:
+      'A peach of a variety with smooth, thin, brightly colored skin and rich firm flesh.',
+  },
+  {
+    name: 'orange',
+    description:
+      'A round juicy citrus fruit with a tough bright reddish-yellow rind',
+  },
+  {
+    name: 'pomegranate',
+    description:
+      'An orange-sized fruit with a tough reddish outer skin and sweet red gelatinous flesh containing many seeds',
+  },
+  {
+    name: 'quince',
+    description:
+      'A hard, acid pear-shaped fruit used in preserves or as flavoring',
+  },
+  {
+    name: 'raspberry',
+    description:
+      'An edible soft fruit related to the blackberry, consisting of a cluster of reddish-pink drupelets',
+  },
+  {
+    name: 'strawberry',
+    description: 'A sweet soft red fruit with a seed-studded surface',
+  },
+  {
+    name: 'tomato',
+    description:
+      'A glossy red, or occasionally yellow, pulpy edible fruit that is eaten as a vegetable or in salad',
+  },
+  {
+    name: 'ugli-fruit',
+    description:
+      'A mottled green and yellow citrus fruit which is a hybrid of a grapefruit and a tangerine',
+  },
+  {
+    name: 'vanilla',
+    description:
+      'A tropical climbing orchid that has fragrant flowers and long podlike fruit',
+  },
+  {
+    name: 'watermelon',
+    description:
+      'The large fruit of a plant of the gourd family, with smooth green skin, red pulp, and watery juice',
+  },
+  {
+    name: 'ximenia',
+    description:
+      'Can be eaten raw and can be used to make juice, jams or intoxicating drinks',
+  },
+  {
+    name: 'yam',
+    description:
+      'The edible starchy tuber of a climbing plant that is widely grown in tropical and subtropical countries',
+  },
+  {
+    name: 'zucchini',
+    description: 'A green variety of smooth-skinned summer squash',
+  },
+];
+
+export const LiveSearch: ComponentStory<typeof SearchInput> = args => {
```

Comment:
When tabbing through menu items, the menu does not scroll to the focus item if its below the fold